tests/foreman/api/test_location.py

```diff
@@ -26,6 +26,7 @@
 from fauxfactory import gen_string
 from requests.exceptions import HTTPError
 
+from robottelo.cli.factory import make_proxy
```

Comment:
No imports ! This should have been called by sat_object like:

`<sat_obj>.cli_factory.make_proxy()`